packages/core/src/interpreter.ts

```diff
@@ -878,6 +880,7 @@ export class Interpreter<
     } else if (isObservable<TEvent>(entity)) {
       return this.spawnObservable(entity, name);
     } else if (isMachine(entity)) {
+      // @ts-ignore
```

Comment:
Is a reason why this is needed to be known? could we add a comment about it?

Reply:
I cannot get this working currently... this was working in the previous TS version so I still need to figure this out.